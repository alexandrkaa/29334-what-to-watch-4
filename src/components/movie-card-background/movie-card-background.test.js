import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardBackground from './movie-card-background.jsx';

it(`<MovieCardBackground /> should match snapshot`, () => {
  const movieBackground = `img/image1.jpg`;
  const title = `Movie title`;
  const tree = renderer
    .create(
        <MovieCardBackground
          movieBackground={movieBackground}
          title={title}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
