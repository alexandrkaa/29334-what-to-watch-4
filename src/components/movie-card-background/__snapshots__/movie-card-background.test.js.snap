// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardBackground /> should match snapshot 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="Movie title"
    src="img/image1.jpg"
  />
</div>
`;
